HW04/anastasia_evsihneyeva/lib/notification.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'singleton'
+
+class Notification
+  attr_reader :log
+
+  def initialize(log = { log: nil })
+    @log = log
+    puts @log[:log]
```

Comment:
```suggestion
    puts @log[:log]
```
it's a bad idea to use it here